packages/menu/test/menu.test.ts

```diff
@@ -707,4 +707,249 @@ describe('Menu', () => {
         await nextFrame();
         expect(el.selected).to.deep.equal(['3']);
     });
+
+    it('prevents selection when scrolling is detected', async () => {
+        const el = await fixture<Menu>(html`
+            <sp-menu selects="single">
+                <sp-menu-item value="1">Item 1</sp-menu-item>
+                <sp-menu-item value="2">Item 2</sp-menu-item>
+            </sp-menu>
+        `);
+        await elementUpdated(el);
+
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const menu = el as any;
```

Comment:
we should avoid using the `any` type, I believe this should be Menu base on the signature above